test/BUILD.gn

```diff
@@ -163,6 +163,7 @@ test("brave_browser_tests") {
     "//brave/browser/ui/webui/brave_new_tab_ui_browsertest.cc",
     "//brave/browser/ui/webui/brave_welcome_ui_browsertest.cc",
     "//brave/common/brave_channel_info_browsertest.cc",
+    "//brave/components/brave_rewards/browser/rewards_service_browsertest.cc",
```

Comment:
I think we want to split out any tests that will make remote network calls because they are subject to intermittent failures that aren't code related

Reply:
cc @bbondy

Reply:
Network calls have been mocked out for this test to mitigate.